scubagoggles/reporter/styles/FrontPageStyle.css

```diff
@@ -51,6 +51,13 @@ table.tenantdata tr:first-child {
 .manual { background-color: var(--test-other); }
 
 .error {
-    background-color: #deb8b8; 
+    background-color: #deb8b8;
     color: #d10000;
-}
\ No newline at end of file
+}
+
+.alignRight {
+    text-align: right;
+    color: grey;
+    font-size: 80%;
+
+}
```

Comment:
The `alignRight` class isn't living up to its name:
![image](https://github.com/user-attachments/assets/a9e799bd-ff0b-4e9d-9c77-e716bcd5acfd)

To achieve the look from the [screenshot](https://github.com/cisagov/ScubaGoggles/issues/439#issuecomment-2435675982) attached to the issue, you could do the following:
```suggestion
.alignRight {
    float: right;
    padding-right: 0.313em;
    color: grey;
    font-size: 80%;

}
```
(borrowed from [here](https://github.com/cisagov/ScubaGoggles/issues/439#issuecomment-2436384536))